check50/__main__.py

```diff
@@ -253,28 +281,76 @@ def raise_invalid_slug(slug, offline=False):
     raise internal.Error(msg)
 
 
+def validate_args(args):
+    """Validate arguments and apply defaults that are dependent on other arguments"""
+
+    # dev implies offline, verbose, and log level "INFO" if not overwritten
+    if args.dev:
+        args.offline = True
+        args.verbose = True
+        if "ansi" in args.output:
+            args.ansi_output = True
+
+    # offline implies local
+    if args.offline:
+        args.no_install_dependencies = True
+        args.no_download_checks = True
+        args.local = True
+
+
+    args.log_level = LogLevel.__members__[args.log_level.upper()]
```

Comment:
I forgot enums implemented `__getitem__`. yeah that's fine. I don't think we really need to check for KeyError since thats already done by argparse as you said. I'm okay with relying on that invariant, but there's no harm in adding it I guess.